src/main/java/de/yuna/berlin/nativeapp/services/http/model/HttpRequest.java

```diff
@@ -0,0 +1,250 @@
+                TypeMap::new
+            ));
+    }
+
+    public static boolean isMethod(HttpRequest request, HttpMethod method) {
+        return request.getMethod().name().equals(method.name());
+    }
+
+    public boolean isMethodGet() {
+        return HttpMethod.GET.equals(method);
+    }
+
+    public boolean isMethodPost() {
+        return HttpMethod.POST.equals(method);
+    }
+
+    public boolean isMethodPut() {
+        return HttpMethod.PUT.equals(method);
+    }
+
+    public boolean isMethodHead() {
+        return HttpMethod.HEAD.equals(method);
+    }
+
+    public boolean isMethodPatch() {
+        return HttpMethod.PATCH.equals(method);
+    }
+
+    public boolean isMethodDelete() {
+        return HttpMethod.DELETE.equals(method);
+    }
+
+    public boolean isMethodOptions() {
+        return HttpMethod.OPTIONS.equals(method);
+    }
+
+    public boolean isMethodTrace() {
+        return HttpMethod.TRACE.equals(method);
+    }
+
+    public boolean isContentType(String contentType) {
+        return headers.getList(HttpHeaders.CONTENT_TYPE).contains(contentType);
+    }
+
+    public boolean isContentTypeJson() {
+        return isContentType(ContentType.APPLICATION_JSON.value());
+    }
+
+    public boolean isContentTypeXml() {
+        return isContentType(ContentType.APPLICATION_XML.value());
+    }
+
+    public boolean isContentTypeXmlSoap() {
+        return isContentType(ContentType.APPLICATION_SOAP_XML.value());
+    }
+
+    public boolean isContentTypeOctetStream() {
+        return isContentType(ContentType.APPLICATION_OCTET_STREAM.value());
+    }
+
+    public boolean isContentTypePdf() {
+        return isContentType(ContentType.APPLICATION_PDF.value());
+    }
+
+    public boolean isContentTypeFormUrlEncoded() {
+        return isContentType(ContentType.APPLICATION_FORM_URLENCODED.value());
+    }
+
+    public boolean isContentTypeMultiPartFormData() {
+        return isContentType(ContentType.MULTIPART_FORM_DATA.value());
+    }
+
+    public boolean isContentTypePlainText() {
+        return isContentType(ContentType.TEXT_PLAIN.value());
+    }
+
+    public boolean isContentTypeHtml() {
+        return isContentType(ContentType.TEXT_HTML.value());
+    }
+
+    public boolean isContentTypeJpeg() {
+        return isContentType(ContentType.IMAGE_JPEG.value());
+    }
+
+    public boolean isContentTypePng() {
+        return isContentType(ContentType.IMAGE_PNG.value());
+    }
+
+    public boolean isContentTypeGif() {
+        return isContentType(ContentType.IMAGE_GIF.value());
+    }
+
+    public boolean isContentTypeMpeg() {
+        return isContentType(ContentType.AUDIO_MPEG.value());
+    }
+
+    public boolean isContentTypeMp4() {
+        return isContentType(ContentType.VIDEO_MP4.value());
+    }
+
+
+    public String bodyAsString() {
+        try {
+            if (body == null)
+                body = new String(exchange.getRequestBody().readAllBytes(), StandardCharsets.UTF_8);
```

Comment:
Use `Charset.defaultCharset()` instead of `StandardCharsets.UTF_8`, so that the user can set its charset by himself in java.